src/main/java/duke/util/TaskList.java

```diff
@@ -0,0 +1,119 @@
+package duke.util;
+
+import java.util.ArrayList;
+
+import duke.DukeException;
+import duke.task.Task;
+
+/**
+ * Represents a list of tasks.
+ */
+public class TaskList {
+    ArrayList<Task> tasks;
```

Comment:
Just a minor point of consideration, personally it may feel a little confusing as 'tasks' and 'task' names are used to represent the ArrayList and individual tasks respectively, perhaps changing the naming for 'tasks' into 'taskList' may improve the distinction of the two variables!